RecoHI/HiMuonAlgos/python/HiRecoMuon_cff.py

```diff
@@ -23,7 +23,14 @@
 
 #don't modify somebody else's sequence, create a new one if needed
 #standalone muon tracking is already done... so remove standalonemuontracking from muontracking
-muonreco_plus_isolation_PbPb = muonreco_plus_isolation.copyAndExclude(standalonemuontracking._seq._collection + displacedGlobalMuonTracking._seq._collection)
+# muonreco_plus_isolation = cms.Sequence(muonreco+muIsolation)
+# muonreco = cms.Sequence(standalonemuontracking+globalmuontracking+muonIdProducerSequence)
+# globalmuontracking = cms.Sequence(globalMuons+tevMuons+displacedGlobalMuonTracking)
+# 
+#muonreco_plus_isolation_PbPb = muonreco_plus_isolation.copyAndExclude(standalonemuontracking+displacedGlobalMuonTracking)
+#eqivalent sequence 
+muonreco_plus_isolation_PbPb = cms.Sequence(globalMuons+tevMuons+muonIdProducerSequence)
```

Comment:
is it possible to maintain the same copyAndExclude logic?
The goal for the original construct was to keep what's in muonreco_plus_isolation except a few.
The current solution means that we will likely diverge in the content if the "upstream" muonreco_plus_isolation is changed and it may take a while to discover.